x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -330,6 +303,27 @@ export function RuleDetailsPage() {
                   itemValue={moment(rule.executionStatus.lastExecutionDate).fromNow()}
                 />
               </EuiFlexGroup>
+              <EuiSpacer size="m" />
+              <EuiFlexGroup>
+                <ItemTitleRuleSummary>
+                  {i18n.translate('xpack.observability.ruleDetails.ruleIs', {
+                    defaultMessage: 'Rule is',
+                  })}
+                </ItemTitleRuleSummary>
+                <EuiFlexItem>
+                  {getRuleStatusDropdown({
```

Comment:
@fkanout This is duplicate code.  It is used in 2 places in this file. Could you possibly create a function that can be reused?